contributing/index.md

```diff
@@ -2,3 +2,70 @@
 title: Contributing
 layout: default
 ---
+
+Zetkin is a big collaboration. This guide is for **your very first hour** of
+learning about it. Welcome!
+
+This is a big document.
+
+## Background: What's Zetkin?
+
+If you are a programmer, user interface designer, user experience designer, or have some other technical competence, you are welcome to contribute to the Zetkin project which would benefit all attached organizations.
+
+Do you want to start contributing right now?
+
+Frontend development in CSS, HTML and JavaScript (React)? You can start right away, on our Open Source projects, using GitHub Issues and Pull Requests to interact with the Zetkin maintainers.
+
+Backend development in Python or Node.js? We do this, but you need to get in touch with the Zetkin maintainers to start doing that.
```

Comment:
Would it make sense to describe the architecture loosely? I'm thinking it might help a new developer to understand that all front-end apps use the Zetkin API which is closed source, and how (or at least _that_) the front-ends when run on their computer will connect to the development API. That's why there is already dummy and stuff available, and why a new front-end contributor does not have to worry about things like that.

It also explains why data might change while they're developing: The same API (and hence the same data) is accessed by other contributors as well!